libs/design-system/src/sidebar/Sidebar.tsx

```diff
@@ -42,7 +42,7 @@ const FooterHolder = styled.div`
   margin-top: auto;
 `;
 
-const COLLAPSED_WIDTH = 480;
+const COLLAPSED_WIDTH = 550;
```

Comment:
to make all channel icons show in a same row in integration store, its a common drawer config, but i dint see any major UI distortion because of this in other part of the app, but let me know if there's a better way.

Reply:
this is causing many integration tests to fail, even though this is not covering the UI cypress complains about it. so i reverted